lib/pundit.rb

```diff
@@ -64,14 +74,19 @@ def verify_policy_scoped
 
   def authorize(record, query=nil)
     query ||= params[:action].to_s + "?"
+
+    if record.blank?
```

Comment:
This will cover a few more objects than `nil`. `'', ' ', nil, [], and {}` e.g. are all blank. I think that's OK here though, the error message below might be misleading however in case something other than `nil` has been passed in. Maybe we can clarify things a bit more in the error message, too, e.g. by saying something like "cannot authorize a blank object".